core/domain/suggestion_registry.py

```diff
@@ -506,11 +511,12 @@ def get_change_list_for_accepting_suggestion(
         old_content = (
             exploration.states[self.change.state_name].content.to_dict())
 
-        # Here, change is of type ExplorationChange and all attributes
-        # on ExplorationChange are created dynamically except cmd, so because
-        # of this MyPy is unable to recognize `old_value` as an attribute
-        # of change and throws an `"ExplorationChange" has no attribute
-        # "old_value"` error. Thus to avoid the error, we used ignore here.
+        # Here we use MyPy ignore because change is of type ExplorationChange
```

Comment:
This should be handled the usual way similar to how you handled ExplorationChange commands. Ditto below.

Reply:
This issue is already fixed in https://github.com/oppia/oppia/pull/15958